src/core/tsi/test_creds/BUILD

```diff
@@ -32,4 +32,6 @@ exports_files([
     "multi-domain.pem",
     "leaf_signed_by_intermediate.key",
     "leaf_and_intermediate_chain.pem",
+    "malformed-cert.pem",
```

Comment:
How did you generate these files? I try to have a README describing how test data is created and ideally a script that can be run to generate it, can you add one or both of those?

e.g. these two :
1. https://github.com/grpc/grpc/blob/fc8ab2b7b454cad2bad42ef18a79d2893524cfc0/test/core/tsi/test_creds/crl_data/README#L1

2. https://github.com/grpc/grpc/blob/fc8ab2b7b454cad2bad42ef18a79d2893524cfc0/test/core/tsi/test_creds/crl_data/intermediate_gen.sh#L1

Reply:
Sure! Added to the README. But note that there is no script to generate this testdata: I just copied one of the existing PEM blocks and deleted a random character. :)